Core/Core/en.lproj/Localizable.strings

```diff
@@ -33,7 +33,7 @@
 "COURSEWARE.IS_FINISHED" = "“ is finished.";
 "COURSEWARE.CONTINUE" = "Continue";
 "COURSEWARE.RESUME" = "Resume";
-"COURSEWARE.CONTINUE_WITH" = "Continue with:";
+"COURSEWARE.RESUME_WITH" = "Resume with:";
```

Comment:
@saeedbashir maybe we need to add this to uk.lproj too (even in English) to don't forget to localise this later. What do you think? Thank you

Reply:
@rnr Yes, it should be changed in all the relevant files. I've changed it where needed.